app/Http/Controllers/ProductsController.php

```diff
@@ -1,29 +1,30 @@
 <?php
 
 namespace App\Http\Controllers;
-
 use App\Product;
 use Illuminate\Http\Request;
 use Illuminate\Support\Facades\File;
 
 class ProductsController extends Controller
+
 {
```

Comment:
Opening brace of a class must be on the line following the class declaration; found 1 line(s)